### DISCUSSION TITLE
`Cannot GET emailverifyxxxxxxxx xxxx [Sep 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3770] [Discord Threads]
> 🙋‍♂️ **Bergi9**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### pls try the latest 0.96.2  
#  
> 🙋‍♂️ **Bergi9**  
#### ok  
#  
> 🙋‍♂️ **Bergi9**  
#### ```   
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **pg**   
#### ProjectOnRootDB: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.96.2**   
#### ```   
#### hm, it seems i already have this version. had it wrong in my head  
#  
> 💁 **o1lab**  
#### ok this may be related to nuxt3 migration that we've done in releases yesterday  
#  
> 💁 **o1lab**  
#### are you behind a reverse proxy ?  
#  
> 🙋‍♂️ **Bergi9**  
#### yes, traefik  
#  
> 💁 **o1lab**  
#### pls raise an issue with all the details  
#  
> 💁 **o1lab**  
#### will take a look  
#  
> 🙋‍♂️ **Bergi9**  
#### ok  
#  
> 🙋‍♂️ **Bergi9**  
#### i resend the user with a new email. he says it works now. thank you for your support  
#  
> 💁 **o1lab**  
#### ahok  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1017431984022229002)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
